pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -2851,6 +2851,22 @@ let
         };
       };
 
+      ms-python.flake8 = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "flake8";
+          publisher = "ms-python";
+          version = "2023.6.0";
```

```suggestion
          version = "2023.13.10681010";
```
I think there is a new version out there. Care to check?

---

Sorry, I'm realizing I don't have the bandwidth to have so many back and forth reviews on this small addition so closing this PR now.

---

https://github.com/microsoft/vscode-flake8/releases/tag/v2023.10.0